app/mod_api/controllers.py

```diff
@@ -171,6 +171,57 @@ def get_metadata(publisher, package):
         return handle_error('GENERIC_ERROR', e.message, 500)
 
 
+@mod_api_blueprint.route("/dataproxy/<publisher>/<package>/<resource>.json", methods=["GET"])
```

Comment:
@subhankarb @rgrp 

why handle json and csv explicitly? What if someone adds a non-tabular data package, or, if at some point we support data package descriptors in yaml etc.? I get that we want to spike a solution and therefore we might base our implementation around *tabular* data package, but the hard coding of format here seems unnecessarily limiting.